### PR TITLE
create-footer

### PR DESCRIPTION
**Pull request by DenysBabenko29**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1 @@
+/*# sourceMappingURL=App.css.map */
```
Comment (Igor-Klein-1988) [inline src/App.css:1]: Why do you use `css`. We use `scss`
why is that?

File: src/features/Footer/AboutUs.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+					<ul className="social__shear d-flex">
+						<li className="social__shear--list">
+							<a
+								className="social__shear--list__icon"
+								target="_blank"
+								href="https://www.facebook.com"
+								rel="noreferrer"
+							>
+								<svg
+									xmlns="http://www.w3.org/2000/svg"
```
Comment (Igor-Klein-1988) [inline src/features/Footer/AboutUs.tsx]: similarly

File: src/features/Footer/AboutUs.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+								</svg>
+								<span className="visually-hidden">Facebook</span>
+							</a>
+						</li>
+						<li className="social__shear--list">
+							<a
+								className="social__shear--list__icon"
+								target="_blank"
+								href="https://twitter.com"
+								rel="noreferrer"
```
Comment (Igor-Klein-1988) [inline src/features/Footer/AboutUs.tsx]: similarly

File: src/features/Footer/AboutUs.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+							>
+								<svg
+									xmlns="http://www.w3.org/2000/svg"
```
Comment (Igor-Klein-1988) [inline src/features/Footer/AboutUs.tsx]: similarly

File: src/features/Footer/AboutUs.tsx [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+								</svg>
+								<span className="visually-hidden">Twitter</span>
+							</a>
+						</li>
+						<li className="social__shear--list">
+							<a
+								className="social__shear--list__icon"
+								target="_blank"
+								href="https://www.instagram.com"
+								rel="noreferrer"
```
Comment (Igor-Klein-1988) [inline src/features/Footer/AboutUs.tsx]: similarly

File: src/features/Footer/AboutUs.tsx [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+							>
+								<svg
+									xmlns="http://www.w3.org/2000/svg"
+									width="16.497"
```
Comment (Igor-Klein-1988) [inline src/features/Footer/AboutUs.tsx]: similarly

File: src/features/Footer/AboutUs.tsx [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+								</svg>
+								<span className="visually-hidden">Instagram</span>
+							</a>
+						</li>
+						<li className="social__shear--list">
+							<a
+								className="social__shear--list__icon"
+								target="_blank"
+								href="https://www.youtube.com"
+								rel="noreferrer"
```
Comment (Igor-Klein-1988) [inline src/features/Footer/AboutUs.tsx]: similarly

File: src/features/Footer/AboutUs.tsx [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+							>
+								<svg
+									xmlns="http://www.w3.org/2000/svg"
+									width="16.49"
```
Comment (Igor-Klein-1988) [inline src/features/Footer/AboutUs.tsx]: similarly

File: src/features/Footer/Categories.tsx [hunk 9]
```diff
@@ -0,0 +1,60 @@
+//import React from 'react';
```
Comment (Igor-Klein-1988) [inline src/features/Footer/Categories.tsx]: delete it

File: src/features/Footer/Categories.tsx [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+const Categories: React.FC = (): JSX.Element => {
+	return (
+		<div className="footer__widget">
+			<h2 className="footer__widget--title text-ofwhite h3">
+				Categories
+				<button className="footer__widget--button" aria-label="footer widget button">
+					<svg
+						className="footer__widget--title__arrowdown--icon"
+						xmlns="http://www.w3.org/2000/svg"
+						width="12.355"
```
Comment (Igor-Klein-1988) [inline src/features/Footer/Categories.tsx]: similarly

File: src/features/Footer/Categories.tsx [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+							transform="translate(-6 -8.59)"
+							fill="currentColor"
+						></path>
+					</svg>
+				</button>
+			</h2>
+			<ul className="footer__widget--menu footer__widget--inner">
+				<li className="footer__widget--menu__list">
+					<a className="footer__widget--menu__text" href="about.html">
+						About Us
```
Comment (Igor-Klein-1988) [inline src/features/Footer/Categories.tsx]: Make consts for all text

File: src/features/Footer/Copyright.tsx [hunk 12]
```diff
@@ -0,0 +1,17 @@
+//import React from 'react';
```
Comment (Igor-Klein-1988) [inline src/features/Footer/Copyright.tsx]: delete

File: src/features/Footer/Footer.tsx [hunk 13]
```diff
@@ -0,0 +1,19 @@
+// import React from 'react';
```
Comment (Igor-Klein-1988) [inline src/features/Footer/Footer.tsx]: delete

File: src/App.css (hunk 1 above)
Comment (DenysBabenko29) [reply src/App.css]: third-party extension added, removed

Comment (DenysBabenko29) [reply src/App.css]: and moved the footer to the folder components

**Review by Igor-Klein-1988 — DISMISSED** [review]

**Review by Igor-Klein-1988 — APPROVED** [review]

**Pull request merged**